Wrappers/Python/cil/optimisation/functions/L1Norm.py

```diff
@@ -187,5 +186,117 @@ def proximal(self, x, tau, out = None):
 
         return soft_shrinkage(x, tau, out = out) 
 
+class WeightedL1Norm(Function):
+    r"""WeightedL1Norm function
+            
+            .. math:: F(x) = ||x||_{\ell^1(w)} (w is array of positive weights)
+    
+    Parameters:
+    -----------
+
+    weight: weight array matching the size of the wavelet coefficients, default None
+        If None returns the regular L1Norm.
+    b: data, default None
+        Translation of the function.
```

Comment:
Not our agreed docstring style!
```suggestion
    weight: ndarray, default None
        Weight array matching the size of the wavelet coefficients. If None returns the regular L1Norm.
    b: DataContainer, default None
        Translation of the function.
```